source/css/scss/_palette.scss

```diff
@@ -141,4 +141,7 @@ $letter-colors: (
 
 .kiva-darkgreen-bg {
 	background-color: $kiva-darkgreen;
-}
\ No newline at end of file
+}
+
+// panel stuff
+$callout-panel-bg: scale-color($vivid-yellow, $lightness: 50%);
```

Comment:
@zvibo - while this is correct and will work fine, I propose a preference for using the Kiva specific color names - in this case $kiva-alert-yellow. Also if the desired color for the panel background is too intense, I'd suggest we add the desired bg color (#fdf5ab ??)to the colors collection (as $kiva-alert-light-yellow ??) and use that instead of applying a manipulation to the $vivid-yellow color.